docs/reference/requirement-sets/office-add-in-requirement-sets.md

```diff
@@ -108,7 +108,7 @@ See [Add-in command requirement sets](add-in-commands-requirement-sets.md).
 
 |**Office hosts**|**Methods in set**|
 |:-----|:-----|
-| Excel on Windows<br>Excel on iPad<br>Excel on Mac<br>OneNote on the web<br>PowerPoint on Windows<br>PowerPoint on the web<br>PowerPoint on iPad<br>PowerPoint on Mac<br>Word 2013 and later on Windows<br>Word 2016 and later on Mac<br>Word on the web<br>Word on iPad|Supports conversion to an image (Office.CoercionType.Image) when writing data using the Document.setSelectedDataAsync method.|
+| See [Image Coercion requirement sets](image-coercion-requirement-sets.md)| Document.setSelectedDataAsync method.|
```

Comment:
Nit: other similar statements in this article have a period at the end of the statement.